src/ILCompiler.ReadyToRun/src/Compiler/DependencyAnalysis/TypeAndMethod.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
```

Comment:
Each source file needs to have the copyright header added. Could you please copy-paste from another file to top here.